pkg/parquetconverter/converter_marker.go

```diff
@@ -0,0 +1,61 @@
+package parquetconverter
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"io"
+	"path"
+
+	"github.com/cortexproject/cortex/pkg/storage/tsdb"
+	"github.com/efficientgo/core/errors"
+	"github.com/go-kit/log"
+	"github.com/oklog/ulid"
+	"github.com/thanos-io/objstore"
+	"github.com/thanos-io/thanos/pkg/runutil"
+)
+
+const (
+	ParquetConverterMakerFileName = "parquet-converter-mark.json"
+	CurrentVersion                = 1
+)
+
+type CompactionMark struct {
+	Version int `json:"version"`
```

Comment:
Maybe we can add a comment about what this version actually means. It is the same as the version of the parquet file schema and updating version will trigger a rewrite of the parquet file?